.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "java.test.config": {
+        "env": {
+            "CLDR_DIR": "${workspaceRoot}",
+            "CLDR_ENVIRONMENT": "UNITTEST"
+        }
+    }
+}
```

Comment:
is this the first file added to the repo that's specific to VS Code? how are you using it?

I've been using VS Code for editing js files, but not for java or actually running any code

Reply:
it enables "run this test" or "debug this test" as a button right next to any test function.